packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.html

```diff
@@ -155,10 +155,7 @@
                                 <a
                                     data-ng-href="{{ ctrl.accountDetails.data.elasticSearchApiUrl }}"
                                     class="oui-tile__button oui-button oui-link_icon oui-button_link"
-                                    data-ng-class="{
-                                        'oui-link_disabled': !(ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0),
-                                        'oui-button_full-width': (ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0)
-                                   }"
+                                    data-ng-class="oui-button_full-width"
```

Comment:
Those changes doesn't look related to the order button, maybe they can be applied in a separate commit

Reply:
Ok, will do an other PR